generate_build_config.py

```diff
@@ -73,11 +86,14 @@ def main():
     parser = argparse.ArgumentParser()
     parser.add_argument('output_filename')
     parser.add_argument('--customisation-script', dest='custom_script')
+    parser.add_argument('--binary-customisation-script',
+                        dest='binary_custom_script')
```

Comment:
README.md has not been updated to reflect the addition of argument flag parsing or these specific changes (with example/explanation of the two script types).